python/lsst/obs/sdss/selectSdssImages.py

```diff
@@ -1,3 +1,5 @@
+from builtins import str
```

Comment:
Should we not have the `import str` line in general, to avoid problems like happened with pipe_tasks? See if removing this breaks anything.